SPECS/kernel/kernel-uki.spec

```diff
@@ -30,7 +32,7 @@ BuildRequires:  kernel = %{version}-%{release}
 BuildRequires:  systemd-ukify
 BuildRequires:  dracut
 BuildRequires:  binutils
-BuildRequires:  systemd-boot
+BuildRequires:  systemd-boot-unsigned
```

Comment:
For now...

Should get clarity on what is needed (in hand) before swapping to signed systemd-boot